vendor/sigs.k8s.io/kustomize/kyaml/filesys/fsondisk_test_unix.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2022 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+//go:build !windows
+// +build !windows
+
+package filesys
+
+import (
+	"path/filepath"
+	"testing"
```

Comment:
🤦‍♀️ I think you're right, I missed this when reviewing that PR... is this a blocker for getting this PR in? 

I can change the filenames immediately, but I'm wondering if we also need to rerelease kustomize so that we can include the change in this PR as well

Reply:
https://github.com/kubernetes-sigs/kustomize/pull/4742

Reply:
yeah, we shouldn't ship binaries that compile in the testing package...

Reply:
These aren't running tests but providing help functions. They should probably be in a `filesystest` package on their own.

Reply:
or change the signature to return string, error and make the test caller deal with the error, which drops the need for this to use the `testing` package at all

Reply:
Rereleasing kustomize with the fix now, this PR will be updated in ~2 hours

Reply:
Updated!